WebARonARKit/WebARonARKit.js

```diff
@@ -562,6 +565,35 @@
       return Array.from(this.planes_.values());
     };
 
+    this.createAnchor = function(modelMatrix) {
+      // Create the js anchor.
+      var anchor = new VRAnchor(modelMatrix);
+      var modelMatrixString = "";
```

Comment:
Could probably use JSON.stringify(anchor) here. Might have to implement some kind of serialize method but would probably be cleaner overall. I leave it up to you, this is fine.

Reply:
Well, the biggest problem is that '[' and ']' will have to be removed as I wanted to do a clean access to the values in the matrix from the native side. I could try and see what happens/what is needed but I think it might get uglier than just this simple for loop. I might be wrong and don't know exactly how you would solve this.

Reply:
it's ok to leave it as-is.